website/content/v1.11/reference/cli.md

```diff
@@ -114,9 +114,51 @@ talosctl cgroups [flags]
 
 * [talosctl](#talosctl)	 - A CLI for out-of-band management of Kubernetes nodes created by Talos
 
+## talosctl cluster create docker
+
+Create a local Docker based kubernetes cluster
+
+```
+talosctl cluster create docker [flags]
+```
+
+### Options
+
+```
+      --config-patch stringArray                 patch generated machineconfigs (applied to all node types), use @file to read a patch from file
+      --config-patch-controlplanes stringArray   patch generated machineconfigs (applied to 'init' and 'controlplane' types)
+      --config-patch-workers stringArray         patch generated machineconfigs (applied to 'worker' type)
+      --cpus-controlplanes string                the share of CPUs as fraction (each control plane/VM) (default "2.0")
+      --cpus-workers string                      the share of CPUs as fraction (each worker/VM) (default "2.0")
+  -p, --exposed-ports string                     comma-separated list of ports/protocols to expose on init node. Ex -p <hostPort>:<containerPort>/<protocol (tcp or udp)>
+  -h, --help                                     help for docker
+      --host-ip string                           Host IP to forward exposed ports to (default "0.0.0.0")
+      --image string                             set a custom image to use (by default derived from  the --talos-version flag)
+      --kubernetes-version string                desired kubernetes version to run (default "1.34.0-rc.1")
+      --memory-controlplanes int                 the limit on memory usage in MB (each control plane/VM) (default 2048)
+      --memory-workers int                       the limit on memory usage in MB (each worker/VM) (default 2048)
+      --mount mount                              attach a mount to the container (docker --mount syntax)
+      --subnet string                            Docker network subnet CIDR (default "10.5.0.0/24")
+      --talos-version string                     the desired Talos version (default "latest")
```

Comment:
I feel it might be more appropriate to actually drop this flag and populate the value of `--image` to be more explicit, i.e. `ghcr.io/siderolabs/talos:v1.11.0` so that people now where the image is, right now it's a bit of a mystery looking at the help